install/install.go

```diff
@@ -660,11 +821,205 @@ func K8sInstaller(c *k8s.Client, o Options) error {
+	}
+
+	var repoFile repo.File
+	repoFile.Update(entry)
+	if err := repoFile.WriteFile(settings.RepositoryConfig, 0644); err != nil {
+		return fmt.Errorf("failed to write repository file: %w", err)
+	}
+
+	client := action.NewUpgrade(actionConfig)
+	client.Namespace = ns
+	client.Timeout = 5 * time.Minute
+	if o.Save {
+		client.DryRun = true
+		client.SkipCRDs = true
+	}
+
+	chartPath, err := client.ChartPathOptions.LocateChart("kubearmor/kubearmor-operator", settings)
+	if err != nil {
+		return fmt.Errorf("failed to locate Helm chart: %w", err)
+	}
+
+	chartRequested, err := loader.Load(chartPath)
+	if err != nil {
+		return fmt.Errorf("failed to load Helm chart: %w", err)
+	}
+	log.SetOutput(io.Discard)
+	upgradeInstaller, err := client.Run(releaseName, chartRequested, values)
+	log.SetOutput(os.Stdout)
+	if err != nil {
+		client.Install = true
+		if client.Install {
+			clientInstall := action.NewInstall(actionConfig)
+			clientInstall.ReleaseName = releaseName
+			clientInstall.Namespace = ns
+			clientInstall.Timeout = 5 * time.Minute
+			clientInstall.CreateNamespace = true
+			if o.Save {
+				clientInstall.DryRun = true
+				clientInstall.SkipCRDs = true
+				clientInstall.ClientOnly = true
+			}
+
+			chartPath, err := clientInstall.ChartPathOptions.LocateChart("kubearmor/kubearmor-operator", settings)
+			if err != nil {
+				return fmt.Errorf("failed to locate Helm chart path: %w", err)
+			}
+
+			chartRequested, err := loader.Load(chartPath)
+			if err != nil {
+				return fmt.Errorf("failed to load Helm chart: %w", err)
+			}
+
+			log.SetOutput(io.Discard)
+			installRunner, err := clientInstall.Run(chartRequested, values)
+			log.SetOutput(os.Stdout)
+			if o.Save {
+				return writeHelmManifests(installRunner.Manifest, "kubearmor.yaml", printYAML, kubearmorConfig)
+			}
+			if err != nil {
+				return fmt.Errorf("failed to install Helm chart: %w", err)
+			}
+			fmt.Println("🛡\tInstalled helm release : " + releaseName)
+		}
+	} else {
+		if o.Save {
+			return writeHelmManifests(upgradeInstaller.Manifest, "kubearmor.yaml", printYAML, kubearmorConfig)
+		}
+		fmt.Println("🛡\tUpgraded Kubearmor helm release : " + releaseName)
+	}
+
+	// Install the CR using operator clientset
+
+	operatorClientSet, err := operatorClient.NewForConfig(c.Config)
+	if err != nil {
+		return fmt.Errorf("failed to create operator client: %w", err)
+	}
+	if _, err := operatorClientSet.OperatorV1().KubeArmorConfigs(ns).Get(context.Background(), "kubearmorconfig-default", metav1.GetOptions{}); err == nil {
+		fmt.Println("ℹ️\tKubeArmorConfig already exists")
+	} else {
+		if _, err := operatorClientSet.OperatorV1().KubeArmorConfigs(ns).Create(context.Background(), kubearmorConfig, metav1.CreateOptions{}); err != nil {
+			return fmt.Errorf("failed to create KubeArmorConfig CR: %w", err)
+		}
+		fmt.Println("😄\tKubeArmorConfig Created")
+	}
+
+	if o.Verify && !o.Save {
+		checkPods(c, o, client.Install)
+	}
+
+	return err
```

Comment:
Not a good practice to return a nil value variable. Let's return nil.
```suggestion
	return nil
```